homeassistant/components/fritzbox/model.py

```diff
@@ -42,3 +47,26 @@ class SwitchExtraAttributes(TypedDict, total=False):
     total_consumption_unit: str
     temperature: str
     temperature_unit: str
+
+
+@dataclass
+class FritzEntityDescription(EntityDescription):
+    """Description for Fritz!Smarthome entities."""
+
+    suitable: Callable[[FritzhomeDevice], bool] | None = None
```

Comment:
If I've seen it correctly, this is basically a required field. Same with `native_value` and `is_on` respectively.
You can use mixins to resolve that. This here is a good example:
https://github.com/home-assistant/core/blob/24d017f9744341d42b8fa837e157846c0bb03707/homeassistant/components/melcloud/sensor.py#L24-L36

As an added benefit, I believe once those are required, a few if statements won't be necessary anymore.